frontend/src/views/main/project/Inspector.vue

```diff
@@ -149,152 +98,144 @@
-    @Prop({required: true}) transformationModifications!: object // used for the variation feedback
-    @Prop({required: true}) inputData!: { [key: string]: string }
-    @Prop({default: false}) isMiniMode!: boolean;
-    loadingData = false;
-    featuresToView: string[] = []
-    errorLoadingMetadata = ""
-    dataErrorMsg = ""
-    classificationResult = null
-    isClassification = isClassification
-    debouncingTimeout: number = 500;
-
-    catalogStore = useCatalogStore()
-
-    async mounted() {
-        await this.loadMetaData();
-    }
-
-    @Watch('originalData')
-    public resetInput() {
-        this.$emit('reset');
-        (this.$refs.dataFormObserver as HTMLFormElement).reset();
-    }
-
-    get inputMetaData() {
-        if (!this.model) {
-            return [];
-        }
-
-        return Object.entries(this.dataset.columnTypes)
-            .map(([name, type]) => ({
-                name,
-                type,
-                values: this.dataset.categoryFeatures ?
-                  this.dataset.categoryFeatures[name] : []
-                  // Provide an empty list in case of null due to DB migration
-            }))
-    }
-
-    @Watch('inputMetaData')
-    async loadMetaData() {
-        this.featuresToView = this.inputMetaData.map(e => e.name)
-    }
-
-    get isInputNotOriginal() { // used in case of opening a feedback where original data and input data passed are different
-        return JSON.stringify(this.inputData) !== JSON.stringify({...this.originalData, ...this.transformationModifications})
-    }
-
-    get textFeatureNames() {
-        return this.inputMetaData.filter(e => e.type == 'text').map(e => e.name)
-    }
-
-    public setResult(r) {
-        if (isClassification(this.model.modelType)) {
-            this.classificationResult = r
-        }
-    }
-
-    async onValuePerturbation(featureMeta) {
-        mixpanel.track("Feature perturbation", {
-            columnType: featureMeta.type,
-            featureName: anonymize(featureMeta.name),
-            modelId: this.model.id,
-            datasetId: this.dataset.id
-        })
-        this.$emit('update:inputData', this.inputData)
-    }
-
-    isFeatureEditable(featureName: string) {
-        if (!this.model.featureNames || this.model.featureNames.length == 0) {
-            // if user doesn't specify feature names consider all columns as feature names
-            return true;
-        }
-        return this.model.featureNames.includes(featureName)
-    }
-
-    get modelFeatures() {
-    return this.inputMetaData
-        .filter(x => (x.name !== this.dataset.target) && (!this.model.featureNames || this.model.featureNames.includes(x.name)))
-        .map(x => x.name);
+
+const loadingData = ref(false);
+const featuresToView = ref<string[]>([])
+const errorLoadingMetadata = ref("")
+const dataErrorMsg = ref("")
+const classificationResult = ref("")
+const debouncingTimeout = ref(500);
```

Comment:
This can be a constant since its immutable `const debouncingTimeout = 500`

Reply:
It's true, I forgot to convert it as a constant when migrating